src/components/uiKit/FilterList/CustomFilterContainer.tsx

```diff
@@ -56,7 +59,15 @@ const CustomFilterContainer = ({
   };
 
   const aggregations = results?.aggregations ? results?.aggregations[filterKey] : {};
-  const filterGroup = getFilterGroup(found, aggregations, [], true);
+  const filterGroup = getFilterGroup(
+    found,
+    aggregations,
+    [],
+    true,
```

Comment:
OK but if possible could you create constants for these? 
```
const YADA_YDADA = []
const IS_BLAH_BLAH = true
...
const filterGroup = getFilterGroup(
    found,
    aggregations,
   YADA_YDADA
    IS_BLAH_BLAH,
    ...
```

Reply:
The component will soon be ported to ferlab-ui, it will be refactored at that moment